content/en/docs/collector/deployment/agent.md

```diff
@@ -6,34 +6,35 @@ weight: 2
 cSpell:ignore: prometheusremotewrite
 ---
 
-The agent collector deployment pattern consists of applications —
-[instrumented][instrumentation] with an OpenTelemetry SDK using [OpenTelemetry
-protocol (OTLP)][otlp] — or other collectors (using the OTLP exporter) that send
-telemetry signals to a [collector][] instance running with the application or on
-the same host as the application (such as a sidecar or a daemonset).
+The agent collector deployment pattern uses applications
+[instrumented][instrumentation] with an OpenTelemetry SDK using the
+[OpenTelemetry Protocol (OTLP)][otlp], or downstream collectors using the OTLP
+exporter. These components send telemetry signals to a [Collector][collector]
+instance running alongside the application or on the same host (for example, as
+a sidecar or a DaemonSet).
```

Comment:
I like that you broke the paragraph into two sentences, but it's still a bit of a mouthful. Let's simplify it a little more. (I know I'm introducing passive voice here, but it's okay in this context.)

```suggestion
In the agent deployment pattern, telemetry signals can come from
- Applications [instrumented][instrumentation] with an OpenTelemetry SDK using the
[OpenTelemetry Protocol (OTLP)][otlp]
- Collectors using the OTLP exporter

The signals are sent to a [Collector][collector]
instance that runs alongside the application or on the same host, such as
a sidecar or a DaemonSet.
```